src/foam/u2/view/TwoFactorAuthenticationVerificationView.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'net.nanopay.sme.ui',
+  name: 'TwoFactorAuthenticationVerificationView',
```

Comment:
Is this really a TwoFactorAuthenticationVerificationView, or just a generic MultiPartView?

Reply:
it is a MultiPartView

Reply:
I'd argue it's just a TwoFactorAuthenticationVerificationView. It's only multiple parts if you restrict "part" to a text input for a single character. We already have `foam.u2.MultiView`, which can be used to bind multiple views to the same data.